modules/project_cleanup/function_source/main.go

```diff
@@ -175,6 +181,19 @@ func checkIfTagKeyShortNameExcluded(shortName string, excludedTagKeys []string)
 	return false
 }
 
+func checkIfCaiFeedsShortNameIncluded(shortName string, includedFeeds []*regexp.Regexp) bool {
```

Comment:
Looks like @daniel-cit also has a similar func in https://github.com/terraform-google-modules/terraform-google-scheduled-function/pull/196/. Could we use a common checkIfNameIncluded(name string, reg []*regexp.Regexp) func?

Reply:
done.